automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -22,7 +22,17 @@ abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAl
   var claimedBillingProject: ClaimedProject = _
   var billingProject : GoogleProject = _
   var ronCluster: Cluster = _
-
+  def enableWelder: Boolean = true
+
+  //To use, comment out the lines in after all that clean-up and run the test once normally. Then, instantiate a mock cluster in your test file via the `mockCluster` method in NotebookTestUtils with the project/cluster created
+  //You must also set debug to true. Example usage (goes in the Spec you are creating):
+  //Consider adding autopause = Some(false) to the cluster request if you encounter issues with autopause 
+  //
+  //debug = true
+  //mockedCluster = mockCluster("gpalloc-dev-master-0h7pzni","automation-test-apm25lvlz")
+  var debug: Boolean = false //if true, will not spin up and tear down a cluster on each test. Used in conjunction with mockedCluster
```

Comment:
These can be vals because they're intended to be immutable. The other fields are vars because they are set in test setup code, which is not the case here.